lib/gooddata/bricks/middleware/logger_middleware.rb

```diff
@@ -28,9 +28,9 @@ def call(params)
         params = params.to_hash
         if params['GDC_LOGGING_OFF']
           logger = NilLogger.new
-        elsif params['log_directory'] && params['execution_id']
-          log_directory = params['log_directory']
-          execution_id = params['execution_id']
+        elsif params['GDC_LOG_DIRECTORY'] && params['GDC_EXECUTION_ID']
```

Comment:
info: Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy) (reek)